### DISCUSSION TITLE
Packaging the Plugin

### DISCUSSION THREAD

**Discussion by Crazytalkbob** [Sixze/ALS-Refactored#510] [General]
Has anyone had success packaging a game using the plugin? I keep running into errors when any game assets reference one of the assets in the Plugins folder.

I've been struggling to find an answer. Everything I find and try seems to fail.

**Comment by MrWilsonWilson** [comment]
Running into something similar where a lot of things referencing ALS character, ALS character movement, some control rig types, give vague errors and "accessed none" or bad cast type, when everything compiles fine, and looks solid in blueprints (and game runs fine) 
Packaging seems to throw some false errors, what gives !

**Comment by xkamronx** [comment]
Not positive of your specific use case issue, but I use ALS Refactored in my shipped game and it packages great. I know of at least 2 other shipped games also using ALS Refactored.

**Comment by GodOfMemes** [comment]
Change the "EngineVersion" section of the UPlugin to your engine version
